src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -129,13 +129,16 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
     isProcessingRedirect,
     stripeSetupIntentId,
     isPaymentSetupSuccessful,
+    paymentSetupErrorCode,
   } = useStripePaymentBySetupIntentId(order.internalID)
 
   // SavingPaymentSpinner is rendered and PaymentContent is hidden when true
   const displayLoading =
     isProcessingRedirect ||
     isSavingPayment ||
-    (!!match?.location?.query?.setup_intent && !bankAccountHasInsufficientFunds)
+    (!!match?.location?.query?.setup_intent &&
+      !bankAccountHasInsufficientFunds &&
+      !paymentSetupErrorCode)
```

Comment:
It's not immediately clear what condition this represents. Can we make a comment about it?